internal/test/wmcb/wmcb_test.go

```diff
@@ -109,6 +109,8 @@ func (vm *wmcbVM) runE2ETestSuite(t *testing.T) {
 	vm.runTestBootstrapper(t)
```

Comment:
Shouldn't we add a test that tries to uninstall the kubelet and asserts that it is handled gracefully when no kubelet service is present?